packages/integration-karma/test/rendering/disconnecting-root-vm/index.spec.js

```diff
@@ -0,0 +1,34 @@
+import { createElement } from 'lwc';
+import ParentCmp from 'x/parent';
+
+describe('disconnecting root vm', () => {
+    // unhandledrejection is not supported in ie11 or firefox. In those browsers we will get false positives
```

Comment:
isn't easier to just install an error boundary as a parent of the root? to be more consistent across browsers in detecting the failure?

Reply:
🤦‍♂ i thought about this while making the repro in the playground, but was adding the errorCallback in the wrong boundary. thanks! that makes things easier!